frontend/src/__fixtures__/state.ts

```diff
@@ -36,4 +36,12 @@ export const state: RootState = {
     tagError: null,
     selectedTagId: null,
   },
+  filters: {
+    selectedProjectIds: [],
```

Comment:
🔖 rename projectIds to fit with backend

Reply:
De manière générale je pense qu'il faut appeler ça des repo partout dans l'app, on filtre vraiment par repo. 🔖